omnibus/config/projects/agent.rb

```diff
@@ -182,13 +182,13 @@
 
 if with_python_runtime? "2"
   dependency 'pylint2'
-  dependency 'datadog-agent-integrations-py2'
 end
 
-if with_python_runtime? "3"
-  dependency 'datadog-agent-integrations-py3'
+if with_python_runtime? "3" or with_python_runtime? "2"
```

Comment:
Is there a case where the overall condition is false?

Reply:
Indeed, but this flavor is in the `iot-agent` project so it should not be an issue for this file

That being said this was mostly a nit/check and definitely not a blocking comment

Reply:
In practice it doesn't make a big difference, since this condition is also checked in the loop inside `datadog-agent-integrations.rb`, so if any of these conditions are actually not met, that definition would run but do pretty much nothing. At any rate, this came to be as a straightforward attempt to maintain the semantics of the original bit of code it replaces.